src/constants/hotkeys.ts

```diff
@@ -18,6 +18,7 @@ const hotkeys = {
   OPEN_EXPLORER_TAB: 'ctrl+shift+e, command+shift+e',
   OPEN_KUSTOMIZATION_TAB: 'ctrl+shift+k, command+shift+k',
   OPEN_HELM_TAB: 'ctrl+shift+h, command+shift+h',
+  OPEN_VALIDATION_TAB: 'ctrl+shift+v, command+shift+v',
```

Comment:
please update the `hotkeys.md` file from the `docs` folder as well to include this new shortcut